src/google/google-calendars/googlecalendarincidencecomparator.h

```diff
@@ -37,7 +37,7 @@
 
 #include "trace.h"
 
-#define SOCIALD_LOG_DEBUG_MAYBE(msg) if (printDebug) SOCIALD_LOG_DEBUG(msg)
+#define SOCIALD_LOG_DEBUG_MAYBE(msg) if (printDebug) qCDebug(lcSocialPlugin) << msg
```

Comment:
It seems strange to retain this one, somehow, but I guess it's fine.

Reply:
This looks like what might come from... In past there used to be requirements like: logging message / args for it / etc should/must not evaluated / passed onwards if the message is eventually not going to be emitted.

As such behavior still makes sense to me - can it somehow be handled via qt logging functionality?

Reply:
Yes the SOCIALD_LOG_DEBUG_MAYBE isn't necessary. Removed and adjusted existing code.